src/Common.Stylecop

```diff
@@ -61,6 +61,15 @@
         </Rule>
       </Rules>
     </Analyzer>
+    <Analyzer AnalyzerId="Microsoft.StyleCop.CSharp.MaintainabilityRules">
+      <Rules>
+        <Rule Name="FileMayOnlyContainASingleClass">
```

Comment:
Yes, there are many cases in RESTier to do so. Shall we follow this to separate those files to contain only one class per file?